src/GHC.hs

```diff
@@ -107,6 +107,8 @@ stage2Packages = return [haddock]
 testsuitePackages :: Action [Package]
 testsuitePackages = return [ checkApiAnnotations
                            , checkPpr
+                           , ghcPkg
```

Comment:
We have ghcPkg in stage1. Do we need it here also?

Reply:
Well, it was not getting built here with `build.sh [...] test`. After adding it there, it was.

Reply:
In this case, maybe there are other stage1 packages that are not built properly. Should we change the need rule implemented there to need all stage1 packages?

Reply:
I think `testsuitePackages` should list all packages that are requried for the testsuite. It doesn't matter what we build in Stage0 or Stage1.

Reply:
Yeah I think it's fine to put those in `testsuitePackages`, as at least for the two ones that I added, they actually are (required and) used by some tests.